docs/tutorials/kafka-migration.rst

```diff
@@ -0,0 +1,368 @@
+
+You can review how to create services integrations here. 
+
+The metric you want to track is called ``kafka_mirrormaker_summary.replication_lag``. The image below showcases a Grafana® dashboard showing the mean MirrorMaker 2 replication lag trending to ``0``.
+
+
+.. image:: /images/tutorials/kafka-migration/mirrormaker-replication-lag.png
+    :alt: Grafana dashboard showing the ``kafka_mirrormaker_summary.replication_lag`` metric
+
+Migrate topic schemas
+---------------------
+
+Apache Kafka topics schemas define the structure of the data in certain topics. They can be migrated in two different methods:
+
+#. By replicating the schemas topic stored in Apache Kafka (usually located in the ``_schemas`` topic).
+#. By extracting the schema information from the source and registering in the target environment using the appropriate APIs.
+
+The second option offers much more control over what schemas are migrated. To register the schemas in an Aiven for Apache Kafka service you can:
+
+* Navigate in the `Aiven Console <https://console.aiven.io/>`__, service page, **Schemas** tab.
+* Use the `Karapace rest APIs <https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceSchemaRegistrySubjectVersionPost>`_.
+* Use :doc:`Aiven command line interface </docs/tools/cli>`.
+
+
+Migrate access control list
+---------------------------
+
+Apache Kafka Access Control Lists (ACLs) define how various users can interact with the topics and schemas. To migrate ACLs we recommend to extract the ACL definition from the source Apache Kafka cluster and recreate them in the target cluster. 
+
+If the target of the migration is Aiven for Apache Kafka you can define the ACLs with:
+
+* The `Aiven Console <https://console.aiven.io/>`__, service page, **Access Control Lists (ACL)** tab.
+* The dedicated `Aiven REST API <https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceKafkaAclAdd>`__.
+* The dedicated :doc:`Aiven CLI command </docs/tools/cli/service/acl>`.
+
+
+Change clients settings
+-----------------------
+
+After the replication flow is running, with the schemas and ACLs are in place, you can then start point producers and consumers to the target Apache Kafka cluster. 
+
+.. Warning::
+
+  To avoid any Apache Kafka messages to be lost during the asynchronous migration performed with MirrorMaker 2, it is suggested to stop the producers, check that both the consumer lag in the source system and the MirrorMaker 2 replication lag is ``0`` and then point producers and consumers to the target Apache Kafka cluster. 
+  
+  The :ref:`migration process <tutorial_kafka_migration_migration_process>` provides a detailed series of steps to follow.
+
+If the target of the migration is Aiven for Apache Kafka, you can find how to download the required certificates and connect as producer or consumer in the :doc:`dedicated documentation </docs/products/kafka/howto/list-code-samples>`.
+
+
+
+.. _tutorial_kafka_migration_migration_process:
```

Comment:
```suggestion
After the replication flow is running and the schemas and ACLs are in place, you can start pointing producers and consumers to the target Apache Kafka cluster. 

.. Warning::

  To avoid losing Apache Kafka messages during the asynchronous MirrorMaker 2 migration, we suggest to stopping the producers, checking that both the consumer lag in the source system and the MirrorMaker 2 replication lag is ``0`` and then pointing producers and consumers to the target Apache Kafka cluster. 
  
  The :ref:`migration process <tutorial_kafka_migration_migration_process>` provides a detailed series of steps to follow.

If the target of the migration is Aiven for Apache Kafka, follow the documentation to :doc:`download the required certificates and connect as a producer or consumer </docs/products/kafka/howto/list-code-samples>`.



.. _tutorial_kafka_migration_migration_process:
```